install_config/registry/deploy_registry_existing_clusters.adoc

```diff
@@ -271,8 +271,9 @@ ifdef::openshift-enterprise[]
 $ oc adm registry --service-account=registry \
     --config=/etc/origin/master/admin.kubeconfig \
     --images='registry.redhat.io/openshift3/ose-${component}:${version}' \
-    --mount-host=<path>
+    --mount-host=<path> <1>
 ----
+<1> Required to pull the correct image for {product-title}. `${component}` and `${version}` are dynamically replaced during installation.
```

Comment:
This option description is as below, it seems different with this description^^
 --mount-host='': If set, the registry volume will be created as a host-mount at this path.

Reply:
Good point, the callout # placement here is confusing, as the description is actually referring to line 273 of the command (the `images` option). I've moved `<3>` up to that line.